src/App.vue

```diff
@@ -42,7 +42,7 @@ watch(modalOpen, val => {
 
 <template>
   <div
-    :class="skin"
+    :class="[{ skin }, userSkin === 'dark-mode' ? 'dark' : '']"
```

Comment:
The object notation for classes is like this: `{ 'class-name': useOrNotBoolean }`
So `{ skin }` becomes `{ skin: !!skin }` and then it adds a `'skin'` class if `skin` is `true`-ish.

What you want is this:

```suggestion
    :class="[ skin, { dark: userSkin === 'dark-mode' }]"
```

In the array `skin` is now simply its value and a `'dark'` class is added if `userSkin === 'dark-mode'`.